src/safeds/data/tabular/containers/_table.py

```diff
@@ -1279,13 +1700,13 @@ def transform_table(self, transformer: TableTransformer) -> Table:
         >>> from safeds.data.tabular.transformation import OneHotEncoder
         >>> from safeds.data.tabular.containers import Table
         >>> transformer = OneHotEncoder()
-        >>> table = Table({"col1": [1, 2, 1], "col2": [1, 2, 4]})
-        >>> fitted_transformer = transformer.fit(table, None)
-        >>> table.transform_table(fitted_transformer)
-           col1__1  col1__2  col2__1  col2__2  col2__4
-        0      1.0      0.0      1.0      0.0      0.0
-        1      0.0      1.0      0.0      1.0      0.0
-        2      1.0      0.0      0.0      0.0      1.0
+        >>> table = Table.from_dict({"a": [1, 2, 1], "b": [1, 2, 4]})
+        >>> transformer = transformer.fit(table, None)
+        >>> table.transform_table(transformer)
+           a__1  a__2  b__1  b__2  b__4
+        0   1.0   0.0   1.0   0.0   0.0
+        1   0.0   1.0   0.0   1.0   0.0
+        2   1.0   0.0   0.0   0.0   1.0
```

Comment:
Not a good example, as the OneHotEncoder shouldn't really be used on numerical values.